### PR TITLE
Make GitHub Discussions link point to Atom Community and remove other support options

### PR DESCRIPTION
**Pull request by softorangetech11**
### Requirements for Contributing Documentation

* Fill out the template below. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* The pull request must only contribute documentation (for example, markdown files or API docs). To contribute other changes, you must use a different template. You can see all templates at <https://github.com/atom/.github/tree/master/.github/PULL_REQUEST_TEMPLATE>.

### Description of the Change
Add the GitHub Discussions link for Atom-Community rather than Atom's repository

### Release Notes
N/A

### CODE REVIEW

**Comment by DeeDeeG** [comment]
I don't know how others feel about this, but the Flight Manual has a ton of useful info in it, and I believe it's not that hard to get our fork of it going. Just have to disable link 404 and 304 checks, and it should post a build. So eventually we want to keep pointing people to the flight manual, just our copy that we can keep updated.

**Review by aminya — CHANGES_REQUESTED** [review]
Those links include the information that is not covered in the discussion link.

File: SUPPORT.md [hunk 1]
```diff
@@ -1,9 +1,6 @@
 # Atom Support
 
-If you're looking for support for Atom there are a lot of options, check out:
-
-* User Documentation &mdash; [The Atom Flight Manual](https://flight-manual.atom.io)
```
Comment (softorangetech11) [inline SUPPORT.md:5]: I'll add the flight manual

**Review by softorangetech11 — COMMENTED** [review]
It's gonna be there

File: SUPPORT.md [hunk 2]
```diff
@@ -1,9 +1,7 @@
 # Atom Support
-
-If you're looking for support for Atom there are a lot of options, check out:
-
+ 
+If you're looking for support for Atom Community you can go to our message board or Atom branded Flight manual linked below:
```
Comment (aminya) [inline SUPPORT.md:3]:
```suggestion
If you're looking for support for Atom Community, you can go to our message board or the Flight manual linked below:
```

**Review by aminya — CHANGES_REQUESTED** [review]

**Review by aminya — APPROVED** [review]

**Pull request merged**